### PR TITLE
[incubator/wekan] Initial commit of chart.

### PR DESCRIPTION
**Pull request by jpds**

### CODE REVIEW

File: incubator/wekan/README.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+|       Parameter         |           Description               |                         Default                     |
+|-------------------------|-------------------------------------|-----------------------------------------------------|
+| `image.pullPolicy`      | Container pull policy               | `IfNotPresent`                                      |
+| `image.repository`      | Container image to use              | `wekanteam/wekan`                                   |
+| `image.tag`             | Container image tag to deploy       | `v0.50`                                             |
+| `wekan.mongoURL`        | MongoDB URL for Wekan               | ``                                                  |
+| `wekan.rootURL`         | Root URL for Wekan                  | ``                                                  |
+| `replicaCount`          | k8s replicas                        | `1`                                                 |
+| `resources.limits.cpu`  | Container requested CPU             | `nil`                                               |
```
Comment (unguiculus) [inline incubator/wekan/README.md:42]: Please align the table nicely.

File: incubator/wekan/templates/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            - containerPort: {{ .Values.service.port }}
+          livenessProbe:
+            httpGet:
+              path: /
+              port: {{ .Values.service.port }}
+          readinessProbe:
+            httpGet:
+              path: /
+              port: {{ .Values.service.port }}
+          env:
```
Comment (unguiculus) [inline incubator/wekan/templates/deployment.yaml:32]: For a consistent list indentation style, please add one level of indentation.

File: incubator/wekan/values.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #  cpu: 100m
+  #  memory: 128Mi
+  # requests:
+  #  cpu: 100m
+  #  memory: 128Mi
+wekan:
+  mongoURL: ''
+  rootURL: ''
```
Comment (unguiculus) [inline incubator/wekan/values.yaml:39]: I'd suggest you go with `mongoUrl` and `rootUrl`.

**Review by unguiculus — CHANGES_REQUESTED** [review]
Thanks for the PR.

**Comment by unguiculus** [comment]
Tests probably fail because of the missing MongoDB. Either add a chart dependency to MongoDB or make sure resources are only created when the URL is set showing a prominent warning via NOTES.txt along with directions how to fix this.

Here's an example: https://github.com/kubernetes/charts/tree/master/stable/datadog

**Comment by unguiculus** [comment]
Marking this as stale. Please update within one week.

**Comment by viglesiasce** [comment]
Closing as stale

**Pull request closed without merging**